src/cljc/rems/common_util.cljc

```diff
@@ -55,7 +55,8 @@
       (is (= "(2)" (andstr "(" (:foo x) ")"))))))
 
 (defn deep-merge [a b]
-  (cond (and (sequential? a)
+  (cond (:replace (meta b)) (merge-with deep-merge b)
```

Comment:
I would rather that we would have a new function deep-replace or sth. Don't quite get why use a function called merge to replace. Imo unnecessary obfuscation.

Reply:
There are more than one way to merge things. For example most deep merge implementations you can find online concatenate lists, whereas this deep merge combines them by index because it’s more useful for the application tests.